jupytercad_freecad/freecad/loader.py

```diff
@@ -150,31 +179,52 @@ def save(self, objects: List, options: Dict, metadata: Dict) -> None:
                 fc_file.addObject(py_obj["shape"], py_obj["name"])
             to_update = [x for x in new_objs if x in current_objs] + to_add
 
+            guidata = options.get("guidata", {})
```

Comment:
As far as I can tell, this is being used for FreeCAD (i.e. `.FCstd` file). 
The variable name might be confusing, should I change it instead?

Reply:
What I mean is that the `guidata` entry in the `options` will not be used by the jupytercad front-end. So setting this entry ([here](https://github.com/jupytercad/JupyterCAD-FreeCAD/blob/c221ecb3b7a0d79c40f5214ff23365ca22c08bb1/jupytercad_freecad/freecad/loader.py#L136)) may not be useful anymore, at least not from the `options`. 

Instead we could keep a `self._guidata` attribute in this class, instead of accessing it through `self._options["guidata"]`. The difference is that `self._options` is part of the shared model so that will be needlessly synchronized between back-end front-end and users.

Reply:
Thanks for the comprehensive clarification 🚀 
This sounds fair, I'll be on it.

Reply:
Done in [`01d3734`](https://github.com/jupytercad/JupyterCAD-FreeCAD/pull/12/commits/01d373436044cd0e838c0d81d93ea8e5284db50f)